sql/core/src/main/scala/org/apache/spark/sql/execution/command/functions.scala

```diff
@@ -140,6 +140,9 @@ case class DescribeFunctionCommand(
             "[WHEN expr4 THEN expr5]* [ELSE expr6] END - " +
             "When `expr1` = `expr2`, returns `expr3`; " +
             "when `expr1` = `expr4`, return `expr5`; else return `expr6`.") :: Nil
+      case "||" =>
```

Comment:
nit: shall we update the above comment in L123 `Hard code "<>", "!=", "between", and "case" ...` as well?

Reply:
Ah, right. I'll update it..